### DISCUSSION TITLE
Bug with sign-up process

### DISCUSSION THREAD

**Discussion by tgrrr** [community/community#42741] [GitHub Education]
Hi,

I am unable to sign up.

I have:
- Submitted a dated school id (the top option in the list)
- Ensured my profile name matches my Full Name. This includes a test with my short name, full name, and with my middle name. It matches my name as it appears on my id card. No dice.
- Information in my bio
- Ensured _Everything_ is in English, this isn't the same problem as this issue: https://github.com/orgs/community/discussions/40895

**Error message:**

    Your application cannot be reviewed until you fix the following:

        You are significantly more likely to be verified if you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name and a short bio.

<img width="732" alt="Screen Shot 2022-12-28 at 1 14 45 pm" src="https://user-images.githubusercontent.com/4689707/209746666-0094dec9-153b-4c5a-a008-61438f7f6a28.png">

**Comment by hiten36** [comment]
Everyone has the opportunity to get this pack and its benefits. Try to edit your github profile - Name, bio, try adding social media links as well (it would help github to identify you), and every field.

There are some discussions on this topic that have been resolved. Here are some links that may help you.

https://github.com/orgs/community/discussions/38918
https://github.com/orgs/community/discussions/30138
https://github.com/orgs/community/discussions/37159

**Comment by tgrrr** [comment]
Hi,

Thanks for your quick response.

RE: https://github.com/orgs/community/discussions/38918
My ID card contained an image of my school

re: https://github.com/orgs/community/discussions/30138
I have used my official email address

re: https://github.com/orgs/community/discussions/37159

- I tried a hard reset
- My uni ID includes my name

**Comment by tgrrr** [comment]
Following up on this request on the holiday break
